lib/rubygems/commands/push_command.rb

```diff
@@ -38,13 +38,18 @@ def initialize
       options[:host] = value
     end
 
+    add_option('--mfa CODE', 'Digit code for multifactor authentication') do |value, options|
```

Comment:
can this repeated code be factored into a module?

Reply:
Yes. Thanks.